integration-cli/docker_cli_info_test.go

```diff
@@ -146,3 +146,21 @@ func (s *DockerSuite) TestInfoDisplaysStoppedContainers(c *check.C) {
 	c.Assert(out, checker.Contains, fmt.Sprintf(" Paused: %d\n", 0))
 	c.Assert(out, checker.Contains, fmt.Sprintf(" Stopped: %d\n", 1))
 }
+
+func (s *DockerSuite) TestInfoDebug(c *check.C) {
```

Comment:
yup - I think you may need to do something like `docker -D run busybox echo hi` and then look for at least one "DEBU" in the output - if there, then it passes.